src/pages/TestPage.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from "react";
+
+import { TableAction } from "../components/data-table/types";
+import ObjectsTable from "../components/data-table/ObjectsTable";
+//@ts-ignore
+import { Typography, Checkbox } from "@material-ui/core";
+
+function createData(
+    name: string,
+    calories: number,
+    fat: number,
+    carbs: number,
+    protein: number
+): any {
```

Comment:
Let TS infer the return type. Also for `headCells`. This is how I noticed that property `sortable` should be optional :)